docs/Configuration/Overview.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Configuration Overview
+permalink: /configuration/overview
+category: Configuration
+menuOrder: 1
+---
+
+Cube.js is designed as an extremely configurable and flexible microservice.
+This flexibility allows you to tune Cube.js to your exact requirements.
+
+There are two main ways you can set configuration options; via a
+configuration file, commonly known as the `cube.js` file; and environment
+variables.
+
+There are some areas of overlap between these options, and we recommend
+checking the configuration precedence. In general, defining options in
+your `cube.js` file is the more flexible option because you can define
+the options using Javascript.
```

Comment:
```suggestion
the options using JavaScript.
```

Surely, I know it's just a typo. Just want to reiterate that we should pay attention to use correct name for technologies and tools.